app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,40 @@
+# require 'pry'
+
+class Api::V1::UsersController < ApplicationController
+  skip_before_action :authenticate_request, only: %i[create login]
+  def index
+    @users = User.all
+    render json: @users
+  end
+
+  def create
+    @user = User.create(user_params)
+    # binding.pry
```

Comment:
- [ ] Kindly, remove this old commented code in order to preserve your repo's cleanliness. 👍